test/bigtest/interactors/job-profiles.js

```diff
@@ -0,0 +1,33 @@
+import {
+  interactor,
+  collection,
+  clickable,
+  blurrable,
+  value,
+  property,
+} from '@bigtest/interactor';
+
+import MultiColumnListInteractor from '@folio/stripes-components/lib/MultiColumnList/tests/interactor';
+
+const CheckboxInteractor = interactor(class CheckboxInteractorImplementation {
```

Comment:
Let's use less bloated syntax OR consider extending `CheckboxInteractor` from `@folio/stripes-components/lib/Checkbox/tests/interactor`
```suggestion
@interactor class CheckboxInteractor {
  static defaultScope = '[class*=checkbox---]';

  clickLabel = clickable('label');
  blurInput = blurrable('input');
  inputValue = value('input');
  isChecked = property('input', 'checked');

  clickAndBlur() {
    return this
      .clickLabel()
      .blurInput();
  }
}
```